client/landing/gutenboarding/components/signup-form/index.tsx

```diff
@@ -158,16 +164,25 @@ const SignupForm = ( { onRequestClose }: Props ) => {
 	}
 
 	const langFragment = lang ? `/${ lang }` : '';
+	const anchorFmParameter = isAnchorFmSignup ? `&anchor_podcast=${ anchorFmPodcastId }` : '';
 	const loginRedirectUrl = encodeURIComponent(
-		`${ window.location.origin }/new${ makePath( Step.CreateSite ) }?new`
+		`${ window.location.origin }/new${
+			isAnchorFmSignup ? makePath( Step.IntentGathering ) : makePath( Step.CreateSite )
+		}?new${ anchorFmParameter }`
 	);
 	const signupUrl = encodeURIComponent( `/new${ makePath( Step[ currentStep ] ) }?signup` );
 	const loginUrl = `/log-in/new${ langFragment }?redirect_to=${ loginRedirectUrl }&signup_url=${ signupUrl }`;
+	const modalTitle = isAnchorFmSignup
+		? 'Create your podcast site with WordPress.com'
+		: 'Save your progress';
+	const modalSubTitle = isAnchorFmSignup
+		? 'Create a WordPress.com account and start creating your free site.'
+		: 'Enter an email and password to save your progress and continue.';
 
 	return (
 		<Modal
 			className={ 'signup-form' }
-			title={ __( 'Save your progress' ) }
+			title={ __( modalTitle ) }
```

Comment:
In PHP those translation functions can't process variables, is that different in JS?
If it's not, feel free to just translate all strings in the ternary above and pass the translated string in the variable.

Reply:
[It's the same](https://github.com/Automattic/i18n-calypso/blob/HEAD/README.md#strings-only), thanks for pointing that out! I [translated the strings in the ternary](https://github.com/Automattic/wp-calypso/pull/47600/commits/a0f9e317771ff52ccc3ce7f6fa2f7c7ecaad1f92) instead.